protocol-units/bridge/contracts/src/NativeBridge.sol

```diff
@@ -19,7 +21,15 @@ contract NativeBridge is AccessControlUpgradeable, PausableUpgradeable, INativeB
 
     mapping(uint256 nonce => OutgoingTransfer) public noncesToOutgoingTransfers;
     mapping(bytes32 bridgeTransferId => uint256 nonce) public idsToIncomingNonces;
+    // mapping(uint256 year => mapping(uint256 month => (uint256 day => uint256 amount))) public outboundRateLimitBudget;
+    // mapping(uint256 year => mapping(uint256 month => (uint256 day => uint256 amount))) public incomingRateLimitBudget;
 
+    uint256 public lastReset;
+    uint256 public outboundRateLimitBudget;
```

Comment:
Budget is not the right term here; these are totals because we are counting. 

These would also need to be bucketed for a sliding window.